### ISSUE TITLE
Fresh install error with Caddy

### ISSUE THREAD

**Issue by cbldev** [intuitem/ciso-assistant-community#563]
**Describe the bug**

A fresh install on Ubuntu with Docker report this error:
`caddy     | Error: adapting config using caddyfile: Caddyfile:2: unrecognized global option: reverse_proxy`

**To Reproduce**
1. clone the repository
2. run `./docker-compose.sh`
3. run `docker compose logs`

**Expected behavior**
A localhost running server

**Environment (please complete the following information):**
 - OS: Ubuntu 22.04.4 LTS

**Comment by cbldev** [comment]
I was able to fix it with this diff:

```diff
--- a/Caddyfile
+++ b/Caddyfile
@@ -1,10 +1,6 @@
-# u24.proxmox.lan is an example for a remote host, adjust to your case when using this variant
-u24.proxmox.lan:8443 {
-    reverse_proxy frontend:3000
-    tls internal
-}
-
-u24.proxmox.lan:9443 {
-    reverse_proxy backend:8000
-    tls internal
+https://localhost:8443 {
+  reverse_proxy /api/iam/sso/redirect backend:8000
+  reverse_proxy /api/accounts/saml/0/acs/ backend:8000
+  reverse_proxy /api/accounts/saml/0/acs/finish/ backend:8000
+  reverse_proxy /* frontend:3000
 }
diff --git a/docker-compose.yml b/docker-compose.yml
index c27326b9..f87c8e6b 100644
--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -27,7 +27,7 @@ services:

   caddy:
     container_name: caddy
-    image: caddy:2.7.6
+    image: caddy:2.8.4
     environment:
       - CISO_ASSISTANT_URL=https://localhost:8443
     depends_on:
@@ -36,11 +36,8 @@ services:
     ports:
       - 8443:8443
     volumes:
-      - ./caddy_data:/data
-    command: |
-      sh -c 'echo $CISO_ASSISTANT_URL "{
-      reverse_proxy /api/iam/sso/redirect backend:8000
-      reverse_proxy /api/accounts/saml/0/acs/ backend:8000
-      reverse_proxy /api/accounts/saml/0/acs/finish/ backend:8000
-      reverse_proxy /* frontend:3000
-      }" > Caddyfile && caddy run'
+      - ./Caddyfile:/etc/caddy/Caddyfile
+      - caddy_data:/data
+
+volumes:
+  caddy_data:
```

**Comment by ab-smith** [comment]
hello @cbldev and thank you !
good catch, no need to go back to previous sample actually but a small regression introduced by the big PR of yesterday. it's fixed in #564
